query-old/docs/api/created-api/hooks.md

```diff
@@ -91,18 +91,40 @@ The query arg is used as a cache key. Changing the query arg will tell the hook
 #### Signature
 
 ```ts
-type MutationHook = () => [
-  (
-    arg: any
-  ) => {
-    unwrap: () => Promise<ResultTypeFrom<D>>;
-  },
-  MutationSubState<D> & RequestStatusFlags
-];
+type UseMutation<Definition> = () => [UseMutationTrigger<Definition>, UseMutationResult<Definition>];
+
+type UseMutationTrigger<Definition> = (
+  arg: ArgTypeFrom<Definition>
+) => Promise<{ data: ResultTypeFrom<Definition> }> & {
```

Comment:
```suggestion
) => Promise<{ data: ResultTypeFrom<Definition> | { error: unknown } }> & {
```
could be a bit more accurate thatn `unknown`, but definitely mention the error here

Reply:
Done, went with this option:

```ts
) => Promise<{ data: ResultTypeFrom<Definition> } | { error: BaseQueryError | SerializedError }> & {
```